src/components/Layout/Filter.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import classes from "../CommonStyles.module.css";
+
+const Filter = (props) => {
```

Comment:
Better to have array of string here received through `props` and then iterate that list inside JSX with `map` function